### ISSUE TITLE
Thinking fast and slow: chapters 33–36: Tue 3rd

### ISSUE THREAD

**Issue by HashNotAdam** [Blackmill/book-club#55]
Aiming to read:
Chapter 33: Reversals
Chapter 34: Frames and Reality
Chapter 35: Two Selves
Chapter 36: Life as a Story

MC: @lachlanhardy
Notes: @antoinemacia

See you 12 pm Tuesday, Mar 3rd @ https://whereby.com/blackmill

Ping gday@blackmill.co if you want a calendar invite and access to the Slack beforehand.

**Comment by HashNotAdam** [comment]
# Discussion notes

While discussing the mock jurors assessing punitive damages for a child who suffered moderate burns and a bank that lost $10 million:
**@mcgain**: has been called up for jury duty twice. The first time he was rejected because of his looks but the second time he was accepted and found that the courts were "much more fair and reasonable than you see on TV".

**@mcgain**: can't help but read the most recent examples in a new context having been set up by the previous chapters.

**@antoinemacia:** Anchoring feels like a subset of framing. "Anchoring applies to numbers and measurements while framing is setting the stage."
**@lachlanhardy:** Within that context, you could be framed by the discussion of the suffering of a burnt child and anchored by the discussion of $10m.

**@lachlanhardy:** The idea of being reality bound vs frame bound was interesting "and by the end of the chapter I was just like... *the world's fucked*"

**@lachlanhardy:** The thought of "losses evokes stronger negative feelings than costs" can be related back to how we discuss Tech Debt. We could discuss the time we will lose when a fragile piece of code inevitably breaks in that frame of losses compared to paying an avoidance cost upfront. This is like paying for a \$5 lottery ticket vs taking a bet that could lose \$5

There was an example in *Frames and Reality* about fairly distributing a child exemption. It was generally agreed that this was extremely difficult to interpret.
**@lachlanhardy:** We often make comment about how many examples in the book revolve around money but this could be an example of how non-monetary examples can be difficult to process.

**@mcgain**: We can see from the studies that when you ask people to opt-out of being an organ donor, rather than opt-in, the participation rate soar. Given the public good this would provide, why don't we do this?
**@lachlanhardy:** Maybe the people who designed the form didn't believe it in?
**@HashNotAdam:** Possibly those who designed the form didn't even know about this study and this is why Nudge Units exist.

**@mcgain**: The topic of *how you treat people equally* is quite interesting. If you look at health care, you could say you will budget the same amount per person so as to treat people equally, however, those in cities will benefit from economies of scale which will result in a better quality of care. It is easy to consider this, however, and much harder to develop a solution.
**@lachlanhardy:** The cartoon of 3 children of different heights trying to watch the football is a great example of equality vs equity—it doesn't really help you evaluate what it means to be equitable in situations like the health care example.
![equity](https://user-images.githubusercontent.com/902714/76278399-b6a95080-62df-11ea-87e9-fef66d7d04a4.jpg)

*Two Selves* introduces an interesting internal conflict between the desire to receive less pain and the idea that sometimes your memory of pain will be reduced after experiencing more of it but with a reduced ending.
**@lachlanhardy:** Do I want my medical procedure to go longer so I forget about how bad it really was? What broader implications would need to be considered when deciding if procedures should be extended?
**@mcgain**: If we accept that the only utility of value is our memory then it could be argued that there is no ethical issue with torture so long as the person doesn't remember it.
🤯🤯🤯
**@HashNotAdam:** While there are bigger implications at play in this specific example, it's not uncommon for us to take unusual paths to make people feel better. Examples include the button to close elevator doors or on pedestrian crossings in the city.

**@lachlanhardy:** is terribly scared by marriage.

**@lachlanhardy:** The idea of the experiencing self not having a voice but the remembering self having a voice that is often wrong resonated. So often our memories can mislead us.
**@mcgain**: There are also implications for delivering a shit sandwich and the tone set at the end of the meeting can influence the pain remembered by people.

**@mcgain**: read that any holiday longer than a week is useless. This could go to the idea that all that matters is the end feeling. You would need to be away long enough to reach peak relaxation or enjoyment but anything after that will not be remembered.
**@HashNotAdam:** has been evaluating taking over 8 months off to travel. Would take a long trip again but much shorter because it's difficult to feel the difference between a 6 and an 8-month trip.

**@mcgain**: has a hypothesis that memories are essentially a Set in that new experiences form new memories but a new memory of the same type does not get stored since it would cause a duplication

**Comment by antoinemacia** [comment]
# Chapter 33 - Reversals

- This chapter goes about the potential incoherence of pretty much anything when they're siloed/taken in isolation
- Depending of the context, the degree of impact varies for the individual/community
- You can see the absurdity/incoherence of an event by viewing the same event in a broader frame

# Chapter 34 - Frames & Reality

- There are various types of framing, which can  transform the perceived *meaning* of words, as well as it's impact on the individual
- Framing is used by various industries using communication to give people augmented feelings in a particular direction, about a given topic (politics, marketing, health & safety measures)
- As system 1 performs the initial unthoughtful reading/listening, most of us passively accept  words as they are framed
- The author prescribes skepticism as a tentative to broaden the picture

# Chapter 35 - Two selves

- Contrast the two different selves of the being, the **remembering self** & the **experiencing self**
- Through various experiments, it is explained that distinguishing memories from experience is a difficult task. What we learn from the past is to maximise the qualities of our future memories, not experience
- While ideas and debate to maximise the feeling of **pleasure**, wether from memories or being in the moment can be a light topic (tourism, activities, mindfulness), discussions around **pain** (the other end of the feeling stick) can be much more difficult and defining the borders of ethics (management of pain in medical, torture...)

# Chapter 36 - Life as a Story

- Basically adds more example and thoughts around the previous chapter.
- Talks about how the remembering self and  its collection of memories are observed to be the person's life as a whole, where the experiencing self a stranger to it
